src/base/bittorrent/session.cpp

```diff
@@ -2777,7 +2777,7 @@ int Session::diskCacheSize() const
 {
     int size = m_diskCacheSize;
     // These macros may not be available on compilers other than MSVC and GCC
-#if defined(__x86_64__) || defined(_M_X64)
+#if (QT_POINTER_SIZE == 8)
```

Comment:
I'd rather define a new macro to have a clear indication of what we want:
```c++
#if (QT_POINTER_SIZE == 8)
#define APP_64BIT
#endif
```
and then:
```c++
#ifdef APP_64BIT
// ...
#else 
// ...
#endif
```

Reply:
How about define it in base/global.h ?

Reply:
>How about define it in base/global.h ?

I've done it anyway, PR updated.